composer.json

```diff
@@ -49,7 +49,8 @@
         "sebastian/version": "^2.0.1"
     },
     "require-dev": {
-        "ext-PDO": "*"
+        "ext-PDO": "*",
+        "vimeo/psalm": "dev-master@dev"
```

Comment:
Not sure what strategy is preferred here in order to verify type checks as part of the CI pipeline. @sebastianbergmann how do you prefer the dev dependency to be used?

Reply:
PHPUnit currently uses a Psalm binary in the `tools` directory, installed via phive and then committed. When I've fixed stuff in Psalm and it's in a release it'll presumably get ported over.

Reply:
Psalm is a tool. I strongly believe that tools must be handled separately from runtime dependencies. Therefore I do not manage tool dependencies with Composer but with Phive. `tools/psalm` is what I use locally and on Travis.

Reply:
Works for me 👍